simulator/simulator/rusEfiFunctionalTest.cpp

```diff
@@ -181,3 +181,6 @@ CANDriver* detectCanDevice(brain_pin_e pinRx, brain_pin_e pinTx) {
 	return &CAND1;
 }
 #endif // HAL_USE_CAN
+
+void setBoardConfigOverrides() {
```

Comment:
does engine_configuration.cpp not get compiled for these targets? This function is defined there as `weak` so it can get replaced by boards that need it.

![image](https://user-images.githubusercontent.com/568254/192089466-0f971024-15cb-4576-bff7-b27502a68859.png)